bots/mlbridge/src/main/java/org/openjdk/skara/bots/mlbridge/ArchiveMessages.java

```diff
@@ -305,6 +305,20 @@ static String composeReplyFooter(PullRequest pr) {
         return "PR: " + pr.webUrl();
     }
 
+    static String composeReplyFooter(PullRequest pr, Comment comment, ReviewComment reviewComment, Review review) {
```

Comment:
I think you should just split this into 3 separate methods. I can't see any caller that uses more than one of Comment, ReviewComment or Review.